### ISSUE TITLE
Outdated waypoint remains on map

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#8237]
**Describe the bug:**
Cache map not updated if waypoints are changed

**To Reproduce:**
1. Open cache
2. Map cache (on new map)
3. Long click on map to create waypoint
4. Save waypoint
5. Map is now shown as intended on the map
6. Click on waypoint
7. Select Edit
8. Enter bearing and distance or just change the coords of the waypoint, save

**Actual behavior/state after performing these steps:**
The cache map does now shows two waypoints (one on old, on on new coords)
Only after restarting the map the outdated waypoint is gone

**Expected behavior/state after performing these steps:**
Edited waypoint should be updated on map

**Version of c:geo used:**
2020.04.20-NB

**Is the problem reproducible:**
Yes (on new map)
Does not happen on GMaps v2

**Comment by geocachermgo** [comment]
ticket #773266

**Comment by geocachermgo** [comment]
@Lineflyer I found this is fixed on 2020.09.07-NB-0542bf7

**Comment by geocachermgo** [comment]
@eddiemuc did you fix it?

**Comment by eddiemuc** [comment]
@geocachermgo I didn't do anything with the map. Maybe a positive side effect of @moving-bits s recent work with routes?

**Comment by geocachermgo** [comment]
Still not ok in all cases (observed it again with 2020.09.09-NB-b8ea9a5)

**Comment by moving-bits** [comment]
Won't fix. Old map implementations are deprecated as of release 2025.07.17 and superseded by UnifiedMap.
